openedx/core/djangoapps/video_pipeline/utils.py

```diff
@@ -1,19 +1,22 @@
+"""
+Utils for video_pipeline app.
+"""
 from django.conf import settings
 from edx_rest_api_client.client import EdxRestApiClient
-
 from openedx.core.lib.token_utils import JwtBuilder
 
 
-def create_video_pipeline_api_client(user, api_url):
+def create_video_pipeline_api_client(user, api_client_id, api_client_secret, api_url):
     """
     Returns an API client which can be used to make Video Pipeline API requests.
 
     Arguments:
         user(User): A requesting user.
         api_url(unicode): It is video pipeline's API URL.
     """
-    jwt_token = JwtBuilder(user).build_token(
+    jwt_token = JwtBuilder(user, secret=api_client_secret).build_token(
         scopes=[],
-        expires_in=settings.OAUTH_ID_TOKEN_EXPIRATION
+        expires_in=settings.OAUTH_ID_TOKEN_EXPIRATION,
+        aud=api_client_id
```

Comment:
what is the purpose of this `aud`?

Reply:
This field `aud` is equivalent to `JWT_AUDIENCE`